src/app/views/main-header/settings/ThemeChooserV9.tsx

```diff
@@ -0,0 +1,88 @@
+import { makeStyles, RadioGroup, Radio} from '@fluentui/react-components';
+import { useAppDispatch, useAppSelector } from '../../../../store';
+import { componentNames, eventTypes, telemetry } from '../../../../telemetry';
+import { PopupsComponent } from '../../../services/context/popups-context';
+import { changeTheme } from '../../../services/slices/theme.slice';
+import { loadGETheme } from '../../../../themes';
+import { translateMessage } from '../../../utils/translate-messages';
+import { BrightnessHighRegular, WeatherMoonFilled, CircleHalfFillFilled } from '@fluentui/react-icons';
+
+const availableThemes = [
+  {
+    key: 'light',
+    displayName: 'Web Light',
+    icon: <BrightnessHighRegular />
+  },
+  {
+    key: 'dark',
+    displayName: 'Web Dark',
+    icon: <WeatherMoonFilled />
+  },
+  {
+    key: 'high-contrast',
+    displayName: 'Teams High Contrast',
+    icon: <CircleHalfFillFilled />
+  }
+];
+
+const useIconOptionStyles = makeStyles({
+  root: {
+    display: 'flex',
+    alignItems: 'center',
+    gap: '5px'
+  },
+  icon: {
+    fontSize: '30px'
+  }
+});
+
+const useLabelStyles = makeStyles({
+  root: {
+    display: 'flex',
+    gap: '5px'
+  }
+});
+
+const ThemeChooserV9: React.FC<PopupsComponent<null>> = () => {
+  const dispatch = useAppDispatch();
+  const appTheme = useAppSelector(state=> state.theme);
+  const iconOptionStyles = useIconOptionStyles();
+  const labelStyles = useLabelStyles();
+
+
+  const handleChangeTheme = (selectedTheme: { key: string; displayName: string; icon: JSX.Element }) => {
+    const newTheme: string = selectedTheme.key;
+    // Applies the theme to the Fluent UI components
+    dispatch(changeTheme(newTheme));
+    loadGETheme(newTheme); //Remove when cleaning up
+    telemetry.trackEvent(eventTypes.BUTTON_CLICK_EVENT, {
+      ComponentName: componentNames.SELECT_THEME_BUTTON,
+      SelectedTheme: newTheme.replace('-', ' ').toSentenceCase()
+    });
+  };
+  return (
+    <RadioGroup layout="horizontal" aria-labelledby='theme-chooser'
+      defaultValue={availableThemes.find(theme => theme.key === appTheme)?.displayName}
```

Comment:
Fantastic work so far!

There's a console warning that draws attention to the implementation you have here
![image](https://github.com/user-attachments/assets/8bcd244f-90e3-4b70-b02f-8952e184b95b)

My understanding is that you can't use `defaultValue` and `checked` at the same time. My recomendation here is to:
1. use the `onChange` listener on `RadioGroup` and remove the `defaultValue` for `value`.
2. You can then create a state `value` which defaults to the `appTheme`, and use all the implementation of `handleChangeTheme` after `setValue(data.value)`. Something like this:
```typescript
 const [value, setValue] = useState(appTheme);
// some stuff
 <RadioGroup value={value} onChange={(_, data) =>{
        const newTheme: string = data.value
        setValue(newTheme)
        // Applies the theme to the Fluent UI components
        dispatch(changeTheme(newTheme));
        loadGETheme(newTheme); //Remove when cleaning up
        telemetry.trackEvent(eventTypes.BUTTON_CLICK_EVENT, {
          ComponentName: componentNames.SELECT_THEME_BUTTON,
          SelectedTheme: newTheme.replace('-', ' ').toSentenceCase()
        });
      }}>
// other stuff
```
3. Remove `checked` and `onClick` on `Radio`.

Borrowed this from https://react.fluentui.dev/?path=/docs/components-radiogroup--docs#controlled-value

Reply:
I'm currently unable to test this locally, but I'm curious if removing the Radio Checked on ln 71 in current implementation would have a similar warning. Per https://react.fluentui.dev/?path=/docs/components-radiogroup--docs#default-value, only either the default Value or the checked can be used